automation/vars/default.yaml

```
@@ -101,15 +101,27 @@ vas:
             src_file: service-values.yaml
         build_output: control-plane.yaml
 
-      - path: examples/va/nfv/sriov/edpm
+      - path: examples/va/nfv/sriov/edpm/nodeset
         wait_conditions:
           - >-
             oc -n openstack wait
-            osdpd edpm-deployment --for condition=Ready --timeout=1200s
+            osdpns openstack-edpm --for condition=SetupReady
+            --timeout=30m
```

it is too much for generating an inventory
```suggestion
            --timeout=10m
```